.changeset/happy-scissors-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+feat: [VS-284] Enable Vectorize query by id via Wrangler
```

Comment:
We should avoid mentioning the jira ticket in the changelog since it is not accessible by the public.
```suggestion
feat: Enable Vectorize query by id via Wrangler
```

Reply:
I've removed the ticket reference in the changelog to prevent noise for our users - the ticket ref will still be in the commit message so hopefully that will be okay for tracking :)